mars/deploy/oscar/cmdline.py

```diff
@@ -80,6 +82,24 @@ def config_args(self, parser):
             "--use-uvloop", help="use uvloop, 'auto' by default. Use 'no' to disable"
         )
 
+    def _set_log_file_env(self):
+        if self.config is None:
+            raise ValueError("Read yml config failed!")
+        cluster_config: dict = self.config.get("cluster")
+        if cluster_config is None:
+            raise KeyError('"cluster" key is missing!')
+        log_dir = cluster_config.get("log_dir")
+        prefix = "mars_"
+        mars_tmp_dir_prefix = "mars_tmp"
+        # default config, then create a temp file
+        if log_dir is None or log_dir == "null":
+            mars_tmp_dir = tempfile.mkdtemp(prefix=mars_tmp_dir_prefix)
```

Comment:
Would it be better to have static methods for temp dir creation and deletion? So that the redundant code in file_logger and UT could be remomved.

Reply:
Fix.